app/models/user.rb

```diff
@@ -1,10 +1,6 @@
-# frozen_string_literal: true
-
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable
   include DeviseTokenAuth::Concerns::User
-
   has_many :rents, dependent: :nullify
+  enum locale: [ :en, :es ]
```

Comment:
We prefer use hash for enum: 
```suggestion
  enum locale: { en: 0, es: 1 }
```